frontend/src/Popover.js

```diff
@@ -29,6 +33,26 @@ export function addPopover(course, courseKey, semIndex) {
     content += "<div class='popover-warning'>This course has previously only been offered in the Spring.</div>";
   }
 
+  let courseId = course["id"];
+  // TODO: Don't hardcode the semester list
+  let courseSemList = ['f19'];
```

Comment:
This should not be hardcoded, especially since some courses don't appear that semester, but when I use
```suggestion
  let courseSemList = course["semester_list"];
```
I get `undefined` for some reason.

@danielleung27 maybe you can figure out how to access the semester list for the course.

Reply:
I think `semester_list` is a property that exists only in the search results course card (`SearchCard`) and not the schedule course card (`CourseCard`). If you want to access it, you can modify the endpoint `get_schedule` in the backend to return it in each course (which is called here to get the schedule course cards: https://github.com/TigerPathApp/tigerpath/blob/38c709d166ff40b7c4030f31a18ec14999aa0a19/frontend/src/components/Schedule.js#L21-L29).